contrib/test_jenkins.sh

```diff
@@ -459,14 +459,34 @@ test_malloc_hooks_mpi() {
 	$MPIRUN -np 1 -x LD_PRELOAD=$ucm_lib $AFFINITY ./test/mpi/test_memhooks -t malloc_hooks
 }
 
+#
+# Run the test suite (gtest) in release configuration
+#
+run_gtest_release() {
+
+	export -n GTEST_SHARD_INDEX
+	export -n GTEST_TOTAL_SHARDS
+	export -n GTEST_SHUFFLE
+	export -n GTEST_TAP
+	export -n GTEST_REPORT_DIR
```

Comment:
why need to unset all these variables?
see example how it should be done in https://github.com/openucx/ucx/blob/master/contrib/test_jenkins.sh#L595
and https://github.com/openucx/ucx/blob/master/contrib/test_jenkins.sh#L605

Reply:
done